index.js

```diff
@@ -18,27 +18,35 @@ var FILTER = filterize([
   /^\._/,
 ]);
 
-function pack(folder) {
-  var tarPack = tar.Pack({ noProprietary: true });
-  var contents = fstream.Reader({
-    path: folder,
-    type: 'Directory',
-    isDirectory: true,
-    filter: FILTER,
+function pack(folder, opts) {
+  opts = opts || {};
```

Comment:
Good catch. Normally I use lodash for this, but didn't want to pull it in when the module is so lightweight